### PR TITLE
Use alternate screen mode. Switch from iostream to stdio.

### PR DESCRIPTION
**Pull request by won3d**
I forked freality and made some simple changes. My first ever attempt at a pull request.

### CODE REVIEW

File: fun/draw.cpp [hunk 1]
```diff
@@ -36,10 +32,16 @@ int setupTerm() {
   if (tcsetattr(0, TCSAFLUSH, &newtermios) < 0) {
     return -1;
   }
+
+  // Switch to alternate screen mode.
+  printf("\033[?1049h");
```
Comment (pablo-mayrgundter) [inline fun/draw.cpp:37]: Neat :)  I kind of like the residue being left, but interesting to take this in more of a first-class app ;)

File: fun/draw.cpp [hunk 2]
```diff
@@ -129,7 +131,7 @@ int readCmd() {
 
 int main() {
   if (!setupTerm()) {
-    cerr << "term setup error" << endl;
+    fputs("term setup error", stderr);
```
Comment (pablo-mayrgundter) [inline fun/draw.cpp:134]: Why prefer stdio?  It was more of a holdover from Damian's.  Just simplicity?

Comment (won3d) [inline fun/draw.cpp:134]: The short answer is that <iostream> is an abomination and you should generally avoid it. Overloading "<<" is idiomatic but totally insane. It is (used to be?) explicitly banned in the Google Style guide with the exception of log messages (for esoteric reasons).

File: fun/draw.cpp (hunk 1 above)
Comment (won3d) [inline fun/draw.cpp:37]: It's also how full screen text apps tend to work, and simplifies a bunch of stuff. I bet you can even skip the "clear" come to think of it.

**Pull request merged**